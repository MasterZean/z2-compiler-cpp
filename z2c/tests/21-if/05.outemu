var a: Int = Int{3}
if (Bool{true})
{
	Int{1}
	Char{49}
}
Double{1}
Double{2}
if (Bool{false}): jumping to instruction 12
Double{3}
Double{4}
