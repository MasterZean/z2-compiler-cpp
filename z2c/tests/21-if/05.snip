def @main() {
	val a = 3
	if (a < 5) {
		1
		'1'
	}
	1.0
	if (3 < 5) {
		2
		'2'
	}
	2.0
	if (a > 5) {
		3
		'3'
	}
	3.0
	if (3 > 5) {
		4
		'4'
	}
	4.0
}