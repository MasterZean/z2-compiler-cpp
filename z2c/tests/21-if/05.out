void _main() {
	int32 a = 3;
	if (a < 5) {
		1;
		'1';
	}
	1.0;
	2.0;
	if (a > 5) {
		3;
		'3';
	}
	3.0;
	4.0;
}

